src/index1.html

```diff
@@ -23,9 +23,9 @@ <h4> Recent Scores</h4>
     <div class="form-cont">
       <h4> Add your score!</h4>
       <form action="#" class="fomr">
-        <input type="text" name="name" placeholder="your name">
-        <input type="number" name="score" placeholder="your score">
-        <input type="submit" value="Submit">
+        <input type="text" name="name" class="inputt" placeholder="your name">
+        <input type="number" name="score" class="inputt" placeholder="your score">
+        <input type="submit" class="btnn input-btn" value="Submit">
```

Comment:
- [ ] Please, consider fixing this form on the top as it is shown on the wireframe. There are too many scores and when I clicked the `refresh` button the score adding form went somewhere down. Let us fix it 👍 

![2022-03-05 21 36 54](https://user-images.githubusercontent.com/49064106/156894052-6cdff9a4-b1ef-4868-84a2-493abe571c81.gif)